pkg/crypto/tinkcrypto/primitive/composite/ecdhes/ecdhes_decrypt_factory.go

```diff
@@ -62,30 +62,48 @@ func newDecryptPrimitiveSet(ps *primitiveset.PrimitiveSet) (*decryptPrimitiveSet
 	return ret, nil
 }
 
+var errNotApplicable = errors.New("not applicable")
+
+func (a *decryptPrimitiveSet) tryNonRawKeys(ct, aad []byte) ([]byte, error) {
```

Comment:
@Baha-sk fyi

Reply:
@soluchok same comment as in `...primitive/composite/ecdh1pu/ecdh1pu_decrypt_factory.go` above